ihatemoney/forms.py

```diff
@@ -393,6 +398,10 @@ def validate_amount(self, field):
             # See https://github.com/python-babel/babel/issues/821
             raise ValidationError(f"Result is too high: {field.data}")
 
+    def validate_bill_type(self, field):
+        if (field.data, field.data) not in Project.bill_types:
+            raise ValidationError(_("Invalid Bill Type"))
+
```

Comment:
I think that you can use a `choices` attribute to the `SelectField` instead of a custom validator.